.github/workflows/api-level-lint.yml

```diff
@@ -10,146 +10,23 @@ on:
       - main
 
 jobs:
-  lint-api-level-abstractions:
+  lint-api-level:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-java@v3
         with:
           distribution: 'adopt'
-          java-version: 18
+          java-version: 17
```

Comment:
I included this change of the Java version since 18 is not an LTS:
https://www.oracle.com/java/technologies/java-se-support-roadmap.html